corenlp/client.py

```diff
@@ -168,11 +169,12 @@ def __init__(self, start_server=True,
         self.default_properties = properties or self.DEFAULT_PROPERTIES
         self.default_output_format = output_format or self.DEFAULT_OUTPUT_FORMAT
 
-    def _request(self, buf, properties):
+    def _request(self, buf, properties, date):
```

Comment:
It appears that the behavior when a date is not provided is not clear. Please set a default value for date (probably should be datetime.now?). Additionally, if the date is set to none, it should probably not be added to the properties dict.

Reply:
When a date is not provided the default value is None (set in annotate and in update). I could set it to the current date with something like this `date = datetime.datetime.now().strftime("%Y-%m-%d")` however I think maybe the default behaviour if you do not submit a date should be that no date is set. This means resulting annotation is identical to the current  implementation.

I agree that there is no point in submitting the date if it has not been set. I will update and resubmit.

thanks

GS